forms/uds/a1/form_a1_ivp_error_checks_p.csv

```diff
@@ -0,0 +1,52 @@
+a1-ivp-p-1001,1001,Alert,a1,I,RACEWHITE,Plausibility,"If one race is known, cannot check Unknown","If  RACEWHITE, RACEBLACK, RACEAIAN, RACENHPI, RACEASIAN, ETHISPANIC, or RACEOTH=1 then RACEUNKN cannot equal 1","If  Q2a. RACEWHITE, Q2b. RACEBLACK, Q2c. RACEAIAN, Q2d. RACENHPI, Q2e. RACEASIAN, Q2f. ETHISPANIC, or Q2g. RACEOTH=1 then Q2h. RACEUNKN cannot equal 1",IF (RACEWHITE=1 or RACEBLACK=1 or RACEAIAN=1 or RACENHPI=1 or RACEASIAN=1 or ETHISPANIC=1 or RACEOTH=1) and RACEUNKN=1,,"RACEWHITE,
+RACEBLACK,
+RACEAIAN,
+RACENHPI,
+RACEASIAN,
+ETHISPANIC,
+RACEOTH,
+RACEUNKN",No,No,
+a1-ivp-p-1002,1002,Alert,a1,I,GENMAN,Plausibility,"If one gender is known, cannot check Don't know","If  GENMAN, GENWOMAN, GENTRMAN, GENTRWOMAN, GENNONBI, GENTWOSPIR, GENOTH, or GENNOANS=1 then GENDKN cannot equal 1","If  Q3a. GENMAN, Q3b. GENWOMAN, Q3c. GENTRMAN, Q3d. GENTRWOMAN, Q3e. GENNONBI, Q3f. GENTWOSPIR, Q3g. GENOTH, or Q3i. GENNOANS=1 then Q3hg. GENDKN cannot equal 1",IF (GENMAN=1 or GENWOMAN=1 or GENTRMAN=1 or GENTRWOMAN=1 or GENNONBI=1 or GENTWOSPIR=1 or GENOTH=1 or GENNOANS=1) and GENDKN=1,,"GENMAN,
+GENWOMAN,
+GENTRMAN,
+GENTRWOMAN, GENNONBI,
+GENTWOSPIR,
+GENOTH,
+GENDKN,
+GENNOANS",No,No,
+a1-ivp-p-1003,1003,Alert,a1,I,GENMAN,Plausibility,"If one gender is known, cannot check No answer","If  GENMAN, GENWOMAN, GENTRMAN, GENTRWOMAN, GENNONBI, GENTWOSPIR, GENOTH, or GENDKN=1 then GENNOANS cannot equal 1","If  Q3a. GENMAN, Q3b. GENWOMAN, Q3c. GENTRMAN, Q3d. GENTRWOMAN, Q3e. GENNONBI, Q3f. GENTWOSPIR, Q3g. GENOTH, or Q3h. GENDKN=1 then Q3i. GENNOANS cannot equal 1",IF (GENMAN=1 or GENWOMAN=1 or GENTRMAN=1 or GENTRWOMAN=1 or GENNONBI=1 or GENTWOSPIR=1 or GENOTH=1 or GENDKN=1) and GENNOANS=1,,"GENMAN,
+GENWOMAN,
+GENTRMAN,
+GENTRWOMAN, GENNONBI,
+GENTWOSPIR,
+GENOTH,
+GENDKN,
+GENNOANS",No,No,
+a1-ivp-p-1004,1004,Alert,a1,I,SEXORNGAY,Plausibility,"If one sexual orientation is known, cannot check Don't know","If  SEXORNGAY, SEXORNHET, SEXORNBI, SEXORNTWOS, SEXORNOTH, or SEXORNNOAN=1 then SEXORNDNK cannot equal 1","If Q6a. SEXORNGAY, Q6b. SEXORNHET, Q6c. SEXORNBI, Q6d. SEXORNTWOS, Q6e. SEXORNOTH, or Q6g. SEXORNNOAN=1 then Q6f. SEXORNDNK cannot equal 1",IF (SEXORNGAY=1 or SEXORNHET=1 or SEXORNBI=1 or SEXORNTWOS=1 or SEXORNOTH=1 or SEXORNNOAN=1) and SEXORNDNK=1,,"SEXORNGAY,
+SEXORNHET,
+SEXORNBI,
+SEXORNTWOS,
+SEXORNOTH,
+SEXORNDNK,
+SEXORNNOAN",No,No,
+a1-ivp-p-1005,1005,Alert,a1,I,SEXORNGAY,Plausibility,"If one sexual orientation is known, cannot check No answer","If  SEXORNGAY, SEXORNHET, SEXORNBI, SEXORNTWOS, SEXORNOTH, or SEXORNDNK=1 then SEXORNNOAN cannot equal 1","If Q6a. SEXORNGAY, Q6b. SEXORNHET, Q6c. SEXORNBI, Q6d. SEXORNTWOS, Q6e. SEXORNOTH, or Q6f. SEXORNDNK=1 then Q6g. SEXORNNOAN cannot equal 1",IF (SEXORNGAY=1 or SEXORNHET=1 or SEXORNBI=1 or SEXORNTWOS=1 or SEXORNOTH=1 or SEXORNDNK=1) and SEXORNNOAN=1,,"SEXORNGAY,
```

Comment:
Please change the var_name for error a1-ivp-p-1005 to "SEXORNNOAN", it's easy to implement this check under that variable.